base/scripts/build_tool.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# build    ... builds a local version
+# run      ... runs the locally built version
+# run-prod ... downloads the version from docker registry and runs it
+# publish  ... publishes the locally built version
+
+set -o errexit; set -o pipefail; set -o nounset
+
+readonly RE_DOCKER_REGISTRY
+readonly RE_DOCKER_PASSWORD
+readonly RE_DOCKER_USERNAME
+
+readonly _BT_MICROSERVICE_NAME="pb-base"
+
+declare _BT_VCS_REF; _BT_VCS_REF="v1.0"
```

Comment:
That's invalid, because that should be a VCS reference.  It is used to label image in line 52 so the image source can always be traced back.

Reply:
As you see on the debian base image, it's not common to add the 'v' prefix to the tag.